src/components/tabController/TabBarItem.tsx

```diff
@@ -0,0 +1,325 @@
+// TODO: support commented props
```

Comment:
Do you think it's possible to fix this file push so it won't be consider as a new file and will replace the original JS file.
If it's too complicated then never mind.. But it'll be nice not to lost git history for this file.

Reply:
Looking in VSCode I have the history, where are you looking?

Reply:
In github? 
This file is considered new, so it's a little difficult to spot what has changed.. 
It's not big deal, but if you can easily fix it, that'll be great..

Reply:
Weird, I did the same thing I did with other files (`git mv <old_file> <new_file>`)...
And I do see the history in VSCode, I think we'll 🤞 for it to be ok?